pkgs/by-name/un/unimatrix/package.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, git
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "unimatrix";
+  version = "unstable-2023-04-25";
+
+  src = fetchFromGitHub {
+    owner = "will8211";
+    repo = pname;
+    rev = "65793c237553bf657af2f2248d2a2dc84169f5c4";
+    hash = "sha256-MtHsoZvUpYPQ5O9CJC7TwcOQOT4ATfyNpoozWYSlI7Q=";
+    # Needed for setuptools-scm to detect version
+    leaveDotGit = true;
```

Comment:
I think this will break things. I tried to use this and the stuff in `.git` can change if new commits are added which will make the hash not match. If this is an active repository they you'll see build errors due to hash mismatch when new commits are added.

i tried to use leaveDotGit and had to remove it because it was causing build breakages.
https://github.com/NixOS/nixpkgs/pull/257452#issuecomment-1742160118